user_tools/src/spark_rapids_pytools/resources/dataproc-configs.json

```diff
@@ -155,40 +155,93 @@
         }
       },
       "supportedGpuInstances": {
-        "n1-standard": {
-          "//description": "N1 standard machine types have 3.75 GB of system memory per vCPU",
-          "software": {},
-          "SysInfo": {},
-          "GpuHWInfo": {},
-          "seriesInfo": {
-            "//description": "describe the sys info based on",
-            "name": "n1-standard-(\\d+)",
-            "vCPUs": [1, 2, 4, 8, 16, 32, 64, 96],
-            "memPerCPU": 3840
-          }
-        },
-        "n1-highmem": {
-          "//description": "N1 high-memory machine types have 6.5 GB of system memory per vCPU.",
-          "software": {},
-          "SysInfo": {},
-          "GpuHWInfo": {},
-          "seriesInfo": {
-            "//description": "describe the sys info based on",
-            "name": "n1-highmem-(\\d+)",
-            "vCPUs": [2, 4, 8, 16, 32, 64, 96],
-            "memPerCPU": 6656
+        "t4": {
+          "GpuHWInfo": {
+            "fullName": "nvidia-tesla-t4"
+          },
+          "instanceList": {
+            "n1-standard": {
+              "//description": "N1 standard machine types have 3.75 GB of system memory per vCPU",
+              "software": {},
+              "SysInfo": {
+                "localSsd": 2,
+                "storageInterface": "scsi"
```

Comment:
Trying to understand storageInterface for "t4". Do we have any instance in "t4" which has "nvme" as storageInterface?

Reply:
Yes we can attach NVME to T4. But we did not recommend any storage interface before. So in that case, it would pickup up default SCSI for T4.

https://cloud.google.com/dataproc/docs/concepts/compute/dataproc-local-ssds